react-components/src/components/Toolbar/Image360HistoricalOverviewToolbar.tsx

```diff
@@ -0,0 +1,110 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { Detail, Flex } from '@cognite/cogs.js';
+import React from 'react';
+import styled from 'styled-components';
+import { Thumbnail } from '../utils/Thumbnail';
+
+export interface Image360RevisionDetails{
+  revisionDate?: string;
+  revisionImageUrl?: string;
+};
+
+export interface Image360HistoricalOverviewToolbarProps{
+  stationId?: string;
+  stationName?: string;
+  collectionId?: string;
+  revisionCollection?: Image360RevisionDetails[];
+};
+
+export const Image360HistoricalOverviewToolbar = ({stationId, stationName, collectionId, revisionCollection}: Image360HistoricalOverviewToolbarProps) => {
+  return(
+    <OverviewContainer>
```

Comment:
On my system (Firefox, Linux), the scrollbar slightly overlaps with the image list, blocking the date for each revision. Is that fixable?

Reply:
https://github.com/cognitedata/reveal/assets/70905152/e2364a00-42d4-403c-8510-49dad90ec876